book/src/faq.md

```diff
@@ -453,8 +454,22 @@ The settings are as follows:
    ```
 
    You can refer to [Redundancy](./redundancy.md) for more information.
-   
-   It is also worth noting that the `--beacon-nodes` flag can also be used for redundancy of beacon nodes. For example, let's say you have a beacon node and a validator client running on the same host, and a second beacon node on another server as a backup. In this case, you can use `lighthouse vc --beacon-nodes http://localhost:5052, http://local_IP:5052` on the validator client.
+
+2. If the beacon node and validator clients are on different servers *and different networks*, the settings are as follows:   
+
+    i. Beacon node:
+
+   The setting is the same as in scenario 1.
```

Comment:
```suggestion
   The setting is the same as in scenario 1.
   
   If the validator client will connect to the beacon node over the internet we strongly recommend that transport-layer encryption (e.g. HTTPS or SSL tunneling) is used to protect the BN/VC connection from malicious actors.
   
   If the beacon node HTTP/HTTPS API is public-facing (i.e. the 5052 port is exposed to the internet) then we strongly recommend users apply IP-address filtering to restrict access to the API. Malicious actors on the internet can cause severe performance issues to exposed beacon node APIs.
```

Reply:
Thanks for the comments, I didn't commit the suggestion directly but revised the doc to include the information.